openreview/venue/venue.py

```diff
@@ -1053,6 +1060,144 @@ def set_SAC_ethics_review_process(self, sac_ethics_flag_duedate=None):
     def open_reviewer_recommendation_stage(self, start_date=None, due_date=None, total_recommendations=7):
         self.invitation_builder.set_reviewer_recommendation_invitation(start_date, due_date, total_recommendations)
 
+    def ithenticate_create_and_upload_submission(self):
+        if not self.iThenticatePlagiarismCheck:
+            raise openreview.OpenReviewException(
+                "iThenticatePlagiarismCheck is not enabled for this venue."
+            )
+
+        self.invitation_builder.set_iThenticate_plagiarism_check_invitation()
+
+        iThenticate_client = openreview.api.iThenticateClient(
+            self.iThenticatePlagiarismCheckApiKey,
+            self.iThenticatePlagiarismCheckApiBaseUrl,
+        )
+
+        submissions = self.get_submissions()
+        for submission in tqdm(submissions):
+            # Create a new submission ID in iThenticate
+            # TODO - Decide what should go in metadata.group_context.owners
+            name = openreview.tools.pretty_id(submission.signatures[0])
+
+            res = iThenticate_client.create_submission(
+                owner=submission.signatures[0],
+                title=submission.content["title"]["value"],
+                timestamp=datetime.datetime.fromtimestamp(
+                    submission.tcdate / 1000, tz=datetime.timezone.utc
+                ).strftime("%Y-%m-%dT%H:%M:%SZ"),
+                owner_first_name=name.split(" ", 1)[0],
+                owner_last_name=name.split(" ", 1)[1],
```

Comment:
if the name has two last names, shouldn't we send both?

Reply:
This will send both since I am splitting it only on the first space
So after the split at index `0` we will have the first name and at index `1` we will have all the last names separated by space.